### ISSUE TITLE
Beispiele

### ISSUE THREAD

**Issue by reifab** [biit-hub/WIO-Terminal-IoTB-Template#3]
Hilfreich wäre ein Ordner mit einem Beispiel. Vorschlag:
Wert eines analogen Eingangs lesen, diesen auf einer Page zyklisch darstellen, Publikation des Wertes über MQTT.

**Comment by biit-hub** [comment]
@reifab 
Braucht es für den Kurs bereits ein Beispiel?
Wäre es nicht besser, wenn ich erst für das neue Konzept Beispiele mache?

**Comment by reifab** [comment]
> @reifab Braucht es für den Kurs bereits ein Beispiel? Wäre es nicht besser, wenn ich erst für das neue Konzept Beispiele mache?

Nein, ich spiele ein eigenes Beispiel (Dokumentiert auf Dokuwiki) durch. Allenfalls kannst Du diese Beispiele dann in das Repo portieren.
